### ISSUE TITLE
Don't work with OpitiFabric/Optifine

### ISSUE THREAD

**Issue by MagneticGolem** [jellysquid3/Lithium#222]
If you start Lithium on Fabric with OptiFabric, even before starting the game, a 'Failed to Launch' screen will appear.
I intend to use both at the same time, since lithium helps a lot in FPS and Optifine makes it possible to use shaders, but even using previous versions of both, none of them seem compatible.
NOTE: In Sodium the same problem occurs with OptiFabric.

![optfabric error](https://user-images.githubusercontent.com/80898609/111628701-6913cd80-87e8-11eb-81d4-9e76a36a102e.png)

**Comment by silversquirl** [comment]
Lithium is purposefully made to disallow loading with optifine due to compatibility issues in the past. Duplicate of #73
